pkg/api/types.go

```diff
@@ -48,6 +48,9 @@ type DeschedulerStrategy struct {
 type StrategyParameters struct {
 	NodeResourceUtilizationThresholds NodeResourceUtilizationThresholds
 	NodeAffinityType                  []string
+	// TopologySpreadConstraints describes how a group of pods should be spread across topology
+	// domains. Descheduler will use these constraints to decide which pods to evict.
+	NamespacedTopologySpreadConstraints []NamespacedTopologySpreadConstraint
```

Comment:
Isn't this name unnecessarily long? It feels a bit like adding part of its documentation to the name.